code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -382,6 +385,7 @@
 /obj/item/udder/Initialize(mapload)
 	create_reagents(50)
 	reagents.add_reagent(/datum/reagent/consumable/milk, 20)
+	src.AddComponent(/datum/component/waddling)
```

Comment:
I don't think the udder itself needs the waddling component.
Probably just the cows.

Reply:
LMAO!  I was trying to make turkeys waddle and didn't look close enough at the path of the proc below it.